src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      method="get"
+      action="/"
+      data-qa="big"
+      class="search"
+    >
+      <label>
+        <img
+        src="images/Search.svg"
+        alt="searc icon"
```

Comment:
There is a typo in the alt attribute value. It should be 'search icon' instead of 'searc icon'.